tests/Unit/RunSingleTest/CMakeLists.txt

```diff
@@ -3,22 +3,48 @@
 
 set(EXECUTABLE "RunSingleTest")
 
+# RunSingleTest can be used to compile only a subset of the code base
+# for testing purposes. It uses the RunTests infrastructure with slightly
+# different implementation in the cpp file in order to not depend on all
+# the test libraries.
+#
+# In order to run the tests in a single file:
```

Comment:
I am confused about the singular/plural nature of...
1. how many tests are compiled. The name `RunSingleTests` implies just one test, but here you say "the tests in a single file", implying multiple tests.
2. how many files are compiled. Here you say "the tests in a single file", implying all the tests should come from one source file, but in the next line you say "the cpp files", implying multiple files.

Can this be clarified (throughout this comment)?